include/dem/pw_contact_force.h

```diff
@@ -80,6 +80,15 @@ class PWContactForce
     return torque_on_walls;
   }
 
+  /** This function is used to update the center of mass container member class
+   * @param Tensor would be added to the member class to make the update
+   */
+  void
+  update_center_mass(const Tensor<1, dim> Tensor)
+  {
+    center_mass_container += Tensor;
```

Comment:
What is this Tensor? Is it the displacement? if so, why not call it displacement?

Reply:
Didn't think about that name too much. I will call it displacement!